src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -873,6 +873,16 @@ private static bool NeedsSeparator(SyntaxToken token, SyntaxToken next)
                 }
             }
 
+            // Require a break between a lambda return type and its open paren. Current token might be:
```

Comment:
>break [](http://example.com/codeflow?start=25&length=5)

Perhaps "space" or "separator" rather than "break", to match other comments and method name.